### ISSUE TITLE
[Quest] The Brothers Bronzebeard

### ISSUE THREAD

**Issue by Kalasbyxan** [wow-mania/Redemption#256]
The quest (http://www.wow-mania.com/armory/?quest=13417) auto-mounts the player onto Brann's aeroplane upon accepting the quest. 

At the moment, Brann disappears and respawns upon accepting the quest, with no mount or ability to complete the quest.

**Comment by wow-mania** [comment]
Fixed for next update (24/08/2016)

**Comment by Calebrimbor** [comment]
Tested it, quest does not work. The url given in the first report seems to lead to a non available quest. It states that it's meant for both factions, but it should be Alliance only.

This is the url what should work:
http://www.wow-mania.com/armory/?quest=12973

I did all the quests from the 2 pre-quest chains, from Sniffing Out The Purpetrator to The Master Explorer before trying. I couldn't accept any quest from Brann's Bronzebeard. When I added the quest he did spawn in front of me, but he didn't auto mount me and I also can't mount the flying machine itself.
